pkg/nrtupdater/nrtupdater.go

```diff
@@ -111,6 +107,12 @@ func (te *NRTUpdater) Update(info MonitorInfo) error {
 	return nil
 }
 
+func (te *NRTUpdater) updateNRTInfo(nrt *v1alpha1.NodeResourceTopology, info MonitorInfo) {
+	nrt.Annotations[AnnotationRTEUpdate] = info.UpdateReason()
+	nrt.TopologyPolicies = []string{te.tmPolicy}
```

Comment:
As mentioned before, this data was derived from a command-line argument.
Who's responsible to update that command-line argument properly? the operator?

Reply:
I don't have a good answer here. It depends on how RTE is deployed. If it is deployed by an operator, then yes, the operator is in charge. However, regardless of this, we need to report the updated value of TM policy should it change for whatever reason. Previously we just didn't do that without no obvious reason to.

Reply:
It could be an operator update or manual update in cases where RTE is manually deployed (e.g. update of the command line argument in RTE deployment specification file followed by kubelet apply). Both these scenarios will benefit from this change.

Reply:
I see the benefit indeed for cases that we can somehow learn Kubelet configuration dynamically (without restarting the pod), but any changes for the command line argument in the RTE deployment will trigger a pod restart, which in turn will update the policy anyway, regardless of this change.

Reply:
thing is that unless someone ALSO deletes the relevant NRT objects, in case of restart the NRT objects will NOT be updated.

Reply:
TIL